.env.example

```diff
@@ -4,7 +4,7 @@ DOMAIN=localhost
 
 # Username and Password for Traefik HTTP Basic Auth
 USERNAME=admin
-HASHED_PASSWORD=$apr1$7UvB4Qa3$9W8H0tmwFbQ9MYljwkbCJ. # password=changethis
+HASHED_PASSWORD='$apr1$7UvB4Qa3$9W8H0tmwFbQ9MYljwkbCJ.' # password=changethis
```

Comment:
Without quotes, docker try to get variable from password (because of the dollar sign)